src/deadline_worker_agent/sessions/actions/scripts/attachment_upload.py

```diff
@@ -0,0 +1,52 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+#! /usr/bin/env python3
+import argparse
+import time
+import os
+import boto3
+
+from deadline.job_attachments import api
+
+"""
+A small script to upload job output. 
+
+Example usage:
+
+python attachment_upload.py \
+    -pm /sessions/session-f63c206fb5f04c04aa17821001aa3847fajfm5x4/path_mapping.json \
+    -s3 s3://test-job-attachment/DeadlineCloud \
+    -m /sessions/session-f63c206fb5f04c04aa17821001aa3847fajfm5x4/manifests/0bb7eb91fdf8780c4a7e6174de6dfc5e_manifest \
+    -m /sessions/session-f63c206fb5f04c04aa17821001aa3847fajfm5x4/manifests/0bb7eb91fdf8780c4a7e6174de6dfc5e_manifest
+"""
+
+
+def upload(s3_root_uri: str, path_mapping_rules: str, manifests: list[str]) -> None:
+    s3_path = f"{os.environ.get("DEADLINE_FARM_ID")}/{os.environ.get("DEADLINE_QUEUE_ID")}/{os.environ.get("DEADLINE_JOB_ID")}/{os.environ.get("DEADLINE_STEP_ID")}/{os.environ.get("DEADLINE_TASK_ID")}/{os.environ.get("DEADLINE_SESSIONACTION_ID")}"
```

Comment:
Super nit, maybe a nice refactoring later to make this a helper function in Job Attachment APIs.

Reply:
Noted!